rspec-mode.el

```diff
@@ -583,6 +583,7 @@ file if it exists, or sensible defaults otherwise."
 
 (defun rspec-spring-p ()
   (and rspec-use-spring-when-possible
+       (not (equal (getenv "SPRING_DISABLE") "1"))
```

Comment:
I suppose so, then. Check all three of these possible values, using `member`.

It's doesn't seem like `false`, etc, should be treated similarly: https://github.com/rails/spring/blob/7efd492018049df43310feb89acbaf226446a123/lib/spring/binstub.rb